src/index.js

```diff
@@ -1,13 +1,132 @@
+// @flow
+
+import { get } from 'lodash';
+
+export type MongoOptionsType = {
+  collectionName: string,
+  timestamps: {
+    createdAt: string,
+    updatedAt: string,
+  }
+};
+
+export type MongoUserObjectType = {
+  username?: string,
+  profile?: Object,
+  services: {
+    password?: {
+      bcrypt: string,
+    },
+  },
+  emails?: [{
+    address: string,
+    verified: boolean,
+  }],
+};
+
+// TODO Will import from @accounts/account once it's published on npm
```

Comment:
No need to wait until publish to npm.. just use git repo instead and once it's published the only change would be on package.json